scripts/setup.d/40site-install

```diff
@@ -1 +1 @@
-drush site-install --site-name=default --account-pass=changeme
+drush site-install --site-name=default --account-name=${DRUPAL_ADMIN_ACCOUNT_NAME} --account-pass=changeme
```

Comment:
On a side note. We should also make the ADMIN_PASS changeable like that through an env variable.